internal/controller/management/projectconfigs/project_configs_test.go

```diff
@@ -279,6 +279,87 @@ func TestReconciler_syncWebhookReceivers(t *testing.T) {
 				)
 			},
 		},
+		{
+			name: "success - quay",
+			reconciler: func() *reconciler {
+				scheme := runtime.NewScheme()
+				require.NoError(t, corev1.AddToScheme(scheme))
+				require.NoError(t, kargoapi.AddToScheme(scheme))
+				return newReconciler(
+					fake.NewClientBuilder().
+						WithScheme(scheme).
+						WithObjects(
+							&kargoapi.ProjectConfig{
+								ObjectMeta: metav1.ObjectMeta{
+									Name:      "fake-name",
+									Namespace: "fake-namespace",
+								},
+								Spec: kargoapi.ProjectConfigSpec{},
+							},
```

Comment:
is there a reason we are using an extra `ProjectConfig` stub with the name "fake-name"? curious if that stub is needed for something specific